### DISCUSSION TITLE
Reduce startup time by check upgrade plugins in longer time

### DISCUSSION THREAD

**Discussion by dlintw** [LazyVim/LazyVim#1452] [Ideas]
Whenever I start nvim, I found it require about 2 seconds or more to check those plugins.  Is there any function can let checking plugin newer version by once per day?

**Comment by dpetka2001** [comment]
Check the [docs](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration) and specifically the section about `checker.frequency`.
